packages/bpk-component-content-cards/src/BpkContentCards.module.scss

```diff
@@ -23,17 +23,18 @@
 @use '../../unstable__bpk-mixins/breakpoints';
 @use '../../unstable__bpk-mixins/typography';
 
-.bpk-content-cards {
-  &--header-text {
-    margin-bottom: tokens.bpk-spacing-base();
+.headerText {
+  margin-top: inherit;
```

Comment:
Is this new?

Reply:
Ah yeah, this is needed to keep the alignment:
![Screenshot 2024-03-21 at 15 00 53](https://github.com/Skyscanner/backpack/assets/7976511/5deaa4e4-d134-4c39-bee4-17c54c654a14)

Reply:
Otherwise it slips down:

![Screenshot 2024-03-21 at 15 01 48](https://github.com/Skyscanner/backpack/assets/7976511/74ad4c7c-94e2-4407-b383-02fad514d919)

Reply:
Should we just set it to a defined value rather than `inherit` or is there a reason `inherit` is needed here?

Reply:
So I think without it, we are getting the browser default for <h2>, but yeah we could use a define value.

With:
![Screenshot 2024-03-21 at 15 07 57](https://github.com/Skyscanner/backpack/assets/7976511/a9be62c0-389f-43c1-b326-73204813f790)

Without:
![Screenshot 2024-03-21 at 15 07 40](https://github.com/Skyscanner/backpack/assets/7976511/2714e23d-06a1-49e4-a6a1-fd6f8b1e2af7)

Reply:
I can set it to 0.

Reply:
Browser default: https://www.w3schools.com/cssref/css_default_values.php
`<h2>` margin top is 0.83rem = 13.28px
We are seeing exactly double that above 26.56px which cannot be coincidental.
Anyway, setting it to 0 reinstates it to what it should be.

Reply:
Aha, nice, I've made the change and pushed it 👍

Reply:
Also now fixed stylelint issues